alf/drivers/policy_driver.py

```diff
@@ -206,11 +206,9 @@ def _step(self, time_step, policy_state):
         step_func = self._algorithm.rollout if self._training else (
             self._algorithm.greedy_predict
             if self._greedy_predict else self._algorithm.predict)
-        policy_step = common.algorithm_step(
-            step_func,
-            self._observation_transformer,
-            time_step,
-            state=policy_state)
+        transformed_time_step = self._algorithm.transform_timestep(time_step)
```

Comment:
It seems that here returning transformed_time_step is only for keeping the shaped rewards info for on_policy_driver. It this is True, I think this additional return adds unnecessary complication to the code structure.

Reply:
It's possible that tranform_timestep may do something else to the time_step in the future. And I cannot think of better way of doing it.